### ISSUE TITLE
Quest 2 custom songs seem to be placed in the wrong folder for the current SR version

### ISSUE THREAD

**Issue by mycenotaph** [tommaier123/NoodleManagerX#13]
Setting up a second Quest 2 and the old folder used to be within the android storage where the 0 byte "PLACE CUSTOM SONGS HERE.txt" file is.

It looks like the custom song folder has been moved to the SynthRidersUC folder structure instead (even though the "PLACE CUSTOM SONGS HERE" file is still in the old location!) so when I tried using noodle manager to move the files over, it dumped them into the old folder and then Synth Riders said there were no custom songs.

I manually copied the files to the correct folder and everything worked fine, it just needs the location updated to work with the current version of Synth Riders

**Comment by tommaier123** [comment]
Are you sure, you were using the NoodleManagerX and not the old NoodleManager? The NoodleManagerX indeed puts songs in the SynthRidersUC folder.

**Comment by mycenotaph** [comment]
![image](https://user-images.githubusercontent.com/121336539/209428502-caf0598b-6cb4-456e-ab58-867ed2c5e7d0.png)

I have used the old Noodle Manager in the past, can't remember which builds but at least a few.

Does it store the location to place files in an appdata folder or the registry or something that it might be pulling from an old version?

**Comment by tommaier123** [comment]
No, the location isn't stored for quest. It should only use the SynthRidersUC folder. Did  you manually select the game folder on the quest?

**Comment by mycenotaph** [comment]
Not sure how to do that, I know the folder for my PCVR is manually edited and in the settings and it seemed to remember where that setting was when I downloaded the new executable and overwrote the version that I already had on the computer, but I don't think I modified it for the quest. It's not connected at this point since I got things loaded by hand, so not sure how to check to see if that setting is pointed at the old folder.

**Comment by tommaier123** [comment]
The folder that is selected shouldn't matter in quest mode. The only way I could see it putting the files in the wrong place is if you select the quest folder in PC mode. So next time you try it make sure the quest icon is shown in the top left.
